pkg/ai/openai.go

```diff
@@ -107,5 +107,5 @@ func (a *OpenAIClient) Parse(ctx context.Context, prompt []string, cache cache.I
 }
 
 func (a *OpenAIClient) GetName() string {
-	return "openai"
+	return Backends[0]
```

Comment:
Adding the indirection to the `Backends` slice doesn't really improve the readability of this , can we revert for all such occurrences in `GetName()` ?